client/src/components/DisplayTable/DisplayTable.tsx

```diff
@@ -76,17 +77,23 @@ const get_column_config_defaults = (index: number) => ({
   is_summable: false,
   is_searchable: false,
   sum_func: (sum: number, value: number) => sum + value,
-  // "raw" value for some complex datasets actually requires processing, as we "raw" values are used for sorting, searching, AND the output csv string
-  raw_formatter: _.identity as (val: CellValue) => string,
   sum_initial_value: 0,
   visibility_toggleable: index !== 0,
+  // "plain" value for some complex datasets actually requires processing, as we use "plain" formatted values in the output csv AND for searching and (by default) sorting
+  plain_formatter: _.identity as (value: CellValue) => string | number,
+  sort_func: (
+    plain_a: string | number,
+    plain_b: string | number,
```

Comment:
Specifically not using `CellValue` type for these, on the off chance that `CellValue` gets more complicated again in the future. Want to be clear that "plain" should be simple string/number values.